react/src/lib/List/index.js

```diff
@@ -426,6 +443,8 @@ List.propTypes = {
   id: PropTypes.string,
   /** @prop Optional tabType prop type to manually set child role | 'listitem' */
   itemRole: PropTypes.string,
+  /** @prop Restricts the traversal of the list with either UP/DOWN, LEFT/RIGHT, or both | 'both' */
+  navigationDirection: PropTypes.oneOf(['vertical', 'horizontal', 'both']),
```

Comment:
Is `both` a common value that we use? Seems like `any` or `all` might be a better choice.

Reply:
I'm not really sure what we use, I feel like any is too vague and all as well.